disco_aws_automation/disco_datapipeline.py

```diff
@@ -438,3 +446,21 @@ def _update_defaults(pipeline_objects, log_location=None, subnet_id=None):
         new_fields[DataPipelineConsts.SUBNET_ID_FIELD] = subnet_id
     if new_fields:
         add_default_object_fields(pipeline_objects, new_fields)
+
+
+def _update_daily_schedule(pipeline_objects, backup_period=None):
+    """
+    Update backup pipeline daily schedule and set backup_period. By default backup
+    period is 1 day
+    :param pipeline_objects:
+    :param backup_period:
+    :return:
+    """
+    if backup_period:
+        for pipeline_object in pipeline_objects:
+            if pipeline_object['id'] == DataPipelineConsts.DAILY_SCHEDULE:
+                fields = pipeline_object['fields']
+                for field in fields:
```

Comment:
Can't you use DictReader for this? https://docs.python.org/2/library/csv.html#csv.DictReader